docs/release-notes.md

```diff
@@ -1,3 +1,21 @@
+## 0.21.0
+
+September 24, 2022
+
+This release replaces the underneath HTTP client used on the `TestClient` (`requests` :arrow_right: `httpx`), and as those clients [differ _a bit_ on their API](https://www.python-httpx.org/compatibility/), your test suite will likely break. To make the migration smoother, you can use the [`bump-testclient`](https://github.com/Kludex/bump-testclient) tool.
+
+### Changed
+* Replace HTTP client on TestClient from `requests` to `httpx` [#1376](https://github.com/encode/starlette/pull/1376).
+
+### Added
+* Add `WebSocketException` and support for WS handlers [#1263](https://github.com/encode/starlette/pull/1263).
```

Comment:
```suggestion
* Add `WebSocketException` and support for WebSocket error handlers [#1263](https://github.com/encode/starlette/pull/1263).
```

Reply:
We differentiate between error and exception handlers, but good point 👍 
```suggestion
* Add `WebSocketException` and support for WebSocket exception handlers [#1263](https://github.com/encode/starlette/pull/1263).
```